pkg/render/guardian.go

```diff
@@ -220,8 +233,11 @@ func (c *GuardianComponent) deployment() client.Object {
 					ServiceAccountName: GuardianServiceAccountName,
 					Tolerations:        append(c.installation.ControlPlaneTolerations, rmeta.TolerateMaster, rmeta.TolerateCriticalAddonsOnly),
 					ImagePullSecrets:   secret.GetReferenceList(c.pullSecrets),
-					Containers:         c.container(),
-					Volumes:            c.volumes(),
+					Containers: []corev1.Container{
```

Comment:
From an architecture perspective I'm very hesitant about adding this to the guardian pod, since at first glance packet capturing has nothing to do with the guardian / voltron tunnel so it doesn't seem to make sense to couple it to this.

Can you explain why it needs to be part of the guardian pod?